lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -48,6 +48,11 @@ static constexpr char kErrorMsgInvalidType[]
 
 namespace {
 
+bool isAttributeValid(const PXR_NS::UsdAttribute& attr)
+{
+    return attr.GetPrim() && attr.GetStage();
```

Comment:
Just curious why we need the attr.GetStage() test.  Shouldn't the attr.GetPrim() test be sufficient?  Is this saying we could have a valid prim that doesn't have a stage?

Reply:
Honestly, I started by writing the check for GetStage() and it was not enough so I added the other. I'll try without it!